core/deps.go

```diff
@@ -0,0 +1,50 @@
+package core
+
+import (
+	"io"
+	"net/http"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+)
+
+func externalHttpSourceToPath(lib string, url string) (path string) {
+	home, _ := os.LookupEnv("HOME")
+	localBase := filepath.Join(home, ".joker-deps", "deps", strings.SplitN(url, "//", 2)[1])
```

Comment:
Joker already uses `.jokerd` directory for linting related files. I think it makes sense to use `.jokerd/deps` for dependencies.